app/templates/praktici_admin.html

```diff
@@ -0,0 +1,146 @@
+            </p>
+        </div>
+    </div>
+</div>
+{% endif %}
+
+{% if session.get('success') %}
+<div class="row">
+    <div class="col">
+        <div class="alert alert-success" role="alert">
+            <p class="mb-0">
+                {{ session.get('success') }}
+            </p>
+        </div>
+    </div>
+</div>
+{% endif %}
+
+<div class="row mb-3">
+    <div class="col-12">
+        <h4>Editace</h4>
+    </div>
+    {% if user %}
+    <div class="col-12">
+        <form action="{{ url_for('view.praktici_admin_edit') }}" name="free-vaccines" method="post">
+            <div class="table-responsive">
+                <table id="centers-table" class="table table-bordered table-striped table-sm">
+                    <thead class="thead-dark">
+                    <tr>
+                        <th scope="col" class="text-center">Vakcína</th>
+                        <th scope="col" class="text-center">Dávek</th>
+                        <th scope="col" class="text-center">Adresa</th>
+                        <th scope="col" class="text-center">Telefon</th>
+                        <th scope="col" class="text-center">Email</th>
+                        <th scope="col" class="text-center">Poznámka</th>
+                    </tr>
+                    </thead>
+                    <tbody>
+                    {% for vaccine in user_vaccines %}
+                    <tr>
+                        <td>
+                            <input type="hidden" name="typ_vakciny[]" value="{{ vaccine.typ_vakciny }}"/>
+                            {{ vaccine.typ_vakciny }}
+                        </td>
+                        <td>
+                            <input type="number" name="pocet_davek[]" value="{{ vaccine.pocet_davek }}" size="4"/>
+                        </td>
+                        <td>
+                            <input type="text" name="adresa[]" value="{{ vaccine.adresa }}" size="25"/>
+                        </td>
+                        <td>
+                            <input type="tel" name="kontakt_tel[]" value="{{ vaccine.kontakt_tel }}" size="9"/>
+                        </td>
+                        <td>
+                            <input type="email" name="kontakt_email[]" value="{{ vaccine.kontakt_email }}" size="20"/>
+                        </td>
+                        <td>
+                            <input type="text" name="poznamka[]" value="{{ vaccine.poznamka }}"/>
+                        </td>
+                    </tr>
+                    {% endfor %}
+                    </tbody>
+                </table>
+            </div>
+            <button type="submit" class="btn btn-primary">Aktualizovat</button>
+            <a class="btn btn-warning" href="{{ url_for('view.praktici_admin_logout') }}">Odhlásit se</a>
+        </form>
+    </div>
+    {% else %}
+    <div class="col-sm-6 mb-3">
+        <div class="card card-body bg-light">
+            <h5>Registrovat se</h5>
```

Comment:
It is necessary to give here a link to https://nrpzs.uzis.cz/ and Detailní záznamy UZIS and Úplný kód without zkrácený kód.
Example:
https://nrpzs.uzis.cz/detail-82892-ordinace-pospisil-s-r-o.html#fndtn-detail_uzis
Maybe notice that the code should be for example:  03496571000000